spacy_transformers/data_classes.py

```diff
@@ -155,11 +155,13 @@ class TransformerData:
     wordpieces: WordpieceBatch
     tensors: List[FloatsXd]
     align: Ragged
+    attention: Optional[tuple] = None  # wanted to make this Optional[Tuple[FloatsXd]],
+    # but got an error I couldn't solve
```

Comment:
Perhaps we need to review/revisit this as well before merging. What was the error you got?

Reply:
I just retried it and ran tests - It works without issue. It is probably from the beginning where I tried to figure the whole thing out. Sadly only recall that I spent a lot of time on that type hint 😅

Reply:
I think it needs to be `Optional[Tuple[FloatsXd, ...]]`. If you only write `Tuple[FloatsXd]`, that means a tuple of length 1, whose item is `FloatsXd`.

(Yes this syntax is cursed.)

Reply:
Ahh yes. It should be fixed now